src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1215,12 +1229,15 @@ var TorrentsTable = new Class({
         var filteredRows = [];
 
         var rows = this.rows.getValues();
+        var filterText = $('torrentsFilterInput').value.trim().toLowerCase();
+        var filterTerms = filterText ? filterText.split(" ") : null;
```

Comment:
the if condition is a bit unclear to me, checking for null? or checking for length == 0?

Reply:
Checking for a length of 0. It can’t be null here because ‘toLowerCase’ returns a string.

We do this because calling split on a blank string returns an array with a single blank element, which is more difficult to check for later.

Reply:
>Checking for a length of 0. 

If so then `(filterText.length > 0)` would be clearer.

Reply:
I'll make it more explicit